tutorials/Window.stopFindInPage.md

```diff
@@ -0,0 +1,14 @@
+Stop findInPage call with the specified action
```

Comment:
This tutorial is not at all descriptive of what the feature does.

Reply:
added better description